### PR TITLE
clarity readme

### PR DESCRIPTION
**Pull request by chengr4**
Improve: https://github.com/rust-lang/rustc-perf/pull/1895#discussion_r1574221832

I am afraid I am not getting this better 😓. Please check it for me.

### CODE REVIEW

File: collector/README.md [hunk 1]
```diff
@@ -515,8 +515,9 @@ versions of `rustc` for individual functions of a single runtime benchmark group
 Codegen diff is currently only implemented for runtime benchmarks.
 
 ## Binary size statistics
-You can use the `binary_stats` command to display size statistics (section and symbol sizes) of
-binary artifacts (executables, libraries). You can compare the binary statistics of:
+You can use the `binary_stats` command to display size statistics (section and symbol sizes) of binary/library.
```
Comment (Kobzol) [inline collector/README.md:518]: I would leave this as it was.

**Review by Kobzol — COMMENTED** [review]
I would revert the first change, the second one looks ok.

File: collector/README.md (hunk 1 above)
Comment (chengr4) [reply collector/README.md]: fixed

**Review by Kobzol — APPROVED** [review]
Thanks :)

**Pull request merged**